cfn-resources/search-deployment/mongodb-atlas-searchdeployment.json

```diff
@@ -0,0 +1,96 @@
+{
+  "additionalProperties": false,
+  "description": "Returns, adds, edits, and removes Atlas Search indexes for the specified cluster. Also returns and updates user-defined analyzers for the specified cluster.",
+  "handlers": {
+    "create": {
+      "permissions": [
+        "secretsmanager:GetSecretValue"
```

Comment:
not sure all secretsmanager permissions are needed apart from read

Reply:
the only permission that is being defined is `secretsmanager:GetSecretValue`, might have confused the fact that this permission is present for all handlers (create/read/update/delete) which is correct

Reply:
sorry, i read it wrong, thx